### PR TITLE
🐛📮Open a bot functionality #78

### LINKED ISSUE #78: 🐛📮 Clicking the bot container does not open the bot
**Describe the bug**

When clicking a bot on the bot dashboard, I should navigate to the bot. 

Currently, only when clicking the "Open bot" button, I navigate to the bot. This navigation should happen when clicking the whole bot, except when clicking the vertical '...'. Clicking the ... should open a user-menu with options

image

### PR DESCRIPTION
**Pull request by Nyash-Mauro**
# Description
When the whole bot was clicked it did not create the bot as intended. I updated the functionality to the whole card to create the bot as seen in the video.

Fixes #78 
## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)

# Checklist:

https://user-images.githubusercontent.com/68531415/194697958-604df2c1-2338-49fd-8af3-6374f87d5478.mp4

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by ReaganCn** [comment]
Please update the PR description accordingly. See example: https://github.com/italanta/elewa/pull/86

File: libs/elements/layout/page/src/lib/components/app-navbar/navbar.component.scss [hunk 1]
```diff
@@ -8,15 +8,15 @@
...
+  background-color: rgba($color: #000000, $alpha: 0.3);
+  cursor: pointer;
+}
 
 .cs-navbar {
   background-color: #fff;
-  border: 1px solid #D1D1D1;
-  color: #3C3C3C;
+  border: 1px solid #d1d1d1;
+  color: #3c3c3c;
```
Comment (ReaganCn) [inline libs/elements/layout/page/src/lib/components/app-navbar/navbar.component.scss:19]: Use theme color variables colors from _variables.scss. You can ask Trevin on how to do it

**Review by ReaganCn — CHANGES_REQUESTED** [review]
Congrats on your first PR. It woould be better if you used a hand cursor when someone hovers over the card to show that it is clickable

**Review by ReaganCn — APPROVED** [review]
Good work!

**Review by Nyash-Mauro — COMMENTED** [review]
Hello I've already made the changes

**Comment by Nyash-Mauro** [comment]
Hello made the changes kindly check on the resolved conflict

**Pull request merged**